### DISCUSSION TITLE
Works with cerbo GX on a seperate Raspi?

### DISCUSSION THREAD

**Discussion by kptkip** [fabian-lauer/dbus-shelly-3em-smartmeter#16] [General]
I hav a cerbo GX and have some raspbnerry Pis in the same network.

Is it possible to use one of the rasPi with your software, use there the IP from the shelly and use the IP of the raspberry in the cerbo GX? Or is it only working with VenusOS on a rasPI?

Thanks Alex

**Comment by fabian-lauer** [comment]
Hi @kptkip  sorry I can't support you here as I run the script on Victrons GX device.
If you are running VenusOS in your PI it should work, but have no experience with this.

**Comment by kptkip** [comment]
Sorry, that was my misunderstanding - triggered by a youtube video, where this script seems to be only possible to use on a raspi with VenusOS.
As I understand it from you, you use it directly on the cerbo GX - this will be also my use-case. Tnx!

**Reply by fabian-lauer** [reply]
Yes - I am directly and only running on Cerbo GX

**Comment by kptkip** [comment]
Is there the possibility to work with two shellies (one for controlling the inverter output and one for the entire system at the grid point)?
